### PR TITLE
Add two.

### PR DESCRIPTION
**Pull request by bangerth**
/rebuild

### CODE REVIEW

**Review by marcfehling — APPROVED** [review]

File: publications-2021.bib [hunk 1]
```diff
@@ -28,6 +28,17 @@ @Article{KourHart2021
...
+author = {Greiner, Alexander and Reiter, Nina and Paulsen, Friedrich and Holzapfel, Gerhard A. and Steinmann, Paul and Comellas, Ester and Budday, Silvia},
+doi = {10.3389/fmech.2021.708350},
+journal = {Frontiers in Mechanical Engineering},
+number = {708350},
+pages = {1--18},
+title = {Poro-Viscoelastic Effects During Biomechanical Testing of Human Brain Tissue},
+url = {https://doi.org/10.3389/fmech.2021.708350},
+volume = {7},
+year = {2021}
 }
```
Comment (Rombur) [inline publications-2021.bib:42]: This `}` belongs to the previous entry

Comment (bangerth) [reply publications-2021.bib:42]: Oops..

**Comment by bangerth** [comment]
Thanks, @Rombur, for pointing out the problem. So fixed!

**Comment by Rombur** [comment]
Looks like CI did not report the status and we can't by-pass the CI to merge. You probably need to rebase to re-trigger the CI.

**Comment by bangerth** [comment]
Hm, there is nothing to rebase on beyond current master :-(

**Comment by marcfehling** [comment]
Try `git commit --amend` which gives your commit and new hash. Then force push.

**Comment by bangerth** [comment]
That did it, thanks for the suggestion!

**Review by kronbichler — APPROVED** [review]

**Pull request merged**